addons/sys_rack/stringtable.xml

```diff
@@ -89,6 +89,11 @@
             <Russian>Использовать</Russian>
             <Turkish>Kullan</Turkish>
         </Key>
+        <Key ID="STR_ACRE_sys_rack_useIntercomRadio">
+            <English>Use via Intercom (Work-Knob to %1)</English>
```

Comment:
```suggestion
            <English>Use via Intercom (WK: %1)</English>
```

Reply:
Neat, even better.

Reply:
or `Work: A` would also work, "WK" is used as an abbreviation for the Work knob so both ways work and are short.

Reply:
Yeah, "Work" is best since that's what the user is going to see on the opened FFCS.